test/integration/examples/bookinfo_test.go

```diff
@@ -6,12 +6,16 @@ package examples
 import (
 	"context"
 	"testing"
+	"runtime"
 
 	"github.com/skupperproject/skupper/test/integration/examples/bookinfo"
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 )
 
 func TestBookinfo(t *testing.T) {
+	if runtime.GOARCH == "s390x" {
```

Comment:
Checking `runtime.GOARCH` is ok for `testLocalGatewayService`, because that's a local test.  For `TestBookInfo`, however, you should be inspecting the cluster's architecture, not the local machine.

Check how that's done for ARM, you can adapt that for Z.